### ISSUE TITLE
ResourceNotFound: airflyd

### ISSUE THREAD

**Issue by archit120** [MikeS96/autonomous_landing_uav#1]
Hey,

This looks like very interesting work! I was trying to run it but I ran into this error. 

```
root@9e777559bcf5:~/src/Firmware# roslaunch mavros_off_board mavros_posix_sitl.launch
...
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/roslaunch/__init__.py", line 306, in main
    p.start()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/roslaunch/parent.py", line 268, in start
    self._start_infrastructure()
...
ResourceNotFound: airflyd
```

Could you help me out?

**Comment by MikeS96** [comment]
Hi @archit120,

I found the problem and it was that the launch file _mavros_posix_sitl_ was pointing to another Launch file that is not listed in the repo. I already corrected that in the last [commit](add4ecf1f760171da91f900fee1653ff9694b93f).

**Comment by MikeS96** [comment]
Hi @archit120 
I found the problem and it was that the launch file _mavros_posix_sitl_ was pointing to another Launch file that was not listed in the repo. I already corrected that in the last commit add4ecf1f760171da91f900fee1653ff9694b93f
